src/main/java/com/apifest/oauth20/tests/RefreshTokenFlowTest.java

```diff
@@ -52,23 +52,24 @@ public void when_refresh_token_valid_then_access_token_issued() throws Exception
         assertTrue(!newAccessToken.contains("{\"error\""));
     }
 
-    @Test
-    public void when_obtain_access_token_with_already_used_refresh_token_return_error() throws Exception {
-        // GIVEN
-        String authCode = obtainAuthCode(clientId, DEFAULT_REDIRECT_URI);
-
-        // WHEN
-        String accessTokenResponse = obtainAccessTokenResponse("authorization_code", authCode, clientId, clientSecret, DEFAULT_REDIRECT_URI);
-        String refreshToken = extractRefreshToken(accessTokenResponse);
-        String newAccessToken = obtainAccessTokenByRefreshToken("refresh_token", refreshToken, clientId, clientSecret, DEFAULT_SCOPE);
-        String superNewAccessToken = obtainAccessTokenByRefreshToken("refresh_token", refreshToken, clientId, clientSecret, DEFAULT_SCOPE);
-
-        // THEN
-        assertNotNull(newAccessToken);
-        assertTrue(!newAccessToken.contains("{\"error\""));
-        assertNotNull(superNewAccessToken);
-        assertTrue(superNewAccessToken.contains("{\"error\""));
-    }
+//    @Test
```

Comment:
I will check this test but if it is not already valid it should be removed not commented - we have VCS for that reason.